### PR TITLE
Ctr 239

### PR DESCRIPTION
**Pull request by vinnicombe**

### CODE REVIEW

File: app/utils/Navigator.scala [hunk 1]
```diff
@@ -71,8 +72,8 @@ class Navigator @Inject()() {
     AnyTaxableInvestmentsId -> selectedTaxableIncomeCheck(NormalMode),
     HowMuchForeignIncomeId -> (_ => routes.AnyTaxableForeignIncomeController.onPageLoad(NormalMode)),
     AnyTaxableForeignIncomeId -> selectedTaxableIncomeCheck(NormalMode),
-    OtherTaxableIncomeNameId -> (_ => routes.HowMuchOtherTaxableIncomeController.onPageLoad(NormalMode)),
-    HowMuchOtherTaxableIncomeId -> (_ => routes.AnyTaxableOtherIncomeController.onPageLoad(NormalMode)),
+//    OtherTaxableIncomeId -> otherTaxableIncome(CheckMode),
```
Comment (garethgooch) [inline app/utils/Navigator.scala]: Are these required for later use?

File: app/controllers/AnyOtherBenefitsController.scala [hunk 2]
```diff
@@ -46,12 +46,9 @@ class AnyOtherBenefitsController @Inject()(appConfig: FrontendAppConfig,
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (authenticate andThen getData andThen requireData) {
     implicit request =>
-      val preparedForm = request.userAnswers.anyOtherBenefits match {
```
Comment (garethgooch) [inline app/controllers/AnyOtherBenefitsController.scala]: Is this not required for reloading the data in?

Comment (vinnicombe) [reply app/controllers/AnyOtherBenefitsController.scala]: User must always answer that question, hence why not prefilling

File: app/utils/Navigator.scala (hunk 1 above)
Comment (vinnicombe) [reply app/utils/Navigator.scala]: Removed

**Pull request merged**